packages/postcss-preset-primer/src/index.js

```diff
@@ -64,6 +65,9 @@ const postcssPresetPrimer = () => {
         }),
       }),
       ...plugins,
+      postcssMixins({
+        mixinsDir: path.join(path.dirname(filepath), 'mixins'),
+      }),
```

Comment:
I think this should come before `...plugins` so that the CSS that it generates also gets transformed by the preset plugins

Reply:
@jonrohan great point, done!